src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -69,6 +76,19 @@ public AlignmentRegion(final GATKRead read) {
         }
     }
 
+    public AlignmentRegion(final Kryo kryo, final Input input) {
+        this.contigId = input.readString();
+        this.assemblyId = input.readString();
+        this.forwardStrandCigar = TextCigarCodec.decode(input.readString());
```

Comment:
`Cigar.toString()` calls `TextCigarCodec.encode()` which is pretty straightforward. `TextCigarCodec.decode()` is slightly less straightforward but still looks like it does what we want. I think it's ok to trust that `TextCigarCodec` is doing what it's supposed to. For clarity I changed line 229 to use `TextCigarCodec.encode()` instead of `toString`.